GCRCatalogs/alphaq.py

```diff
@@ -105,7 +105,7 @@ def _collect_native_quantities(name, obj):
                               'magnification',
                               'shear1',
                               'shear2'),
-            'magnification': (lambda mag: np.where(mag<0.1,np.ones_like(mag),mag),
+            'magnification': (lambda mag: np.where(mag<0.1, np.ones_like(mag), mag),
```

Comment:
`np.where` can broadcast, so no need to first allocate memory for 1's. 
```python
lambda mag: np.where(mag < 0.1, 1.0, mag),
```
BTW, this is changing all magnification less than 0.1 to 1. Is this intended?

Reply:
Yes, it is intended. These small values (<0.1) are unphysical and have been traced to edge effects in the lensing code (all galaxies with these small values fall clode to the boundary of the cut-out for the catalog. It is reasonable to set these v

Reply:
Ok, I'm just making sure since this change also implies that magnification cannot be less than 0.1.